docs/how-to-build-with-ddn/with-sqlserver.mdx

```diff
@@ -0,0 +1,453 @@
+values:
+
+**Connection string**
+
+The connection string format will be in the format
+`Server=<hostname>,<port>;Database=<database>;Uid=<username>;Password=<password>` and so is as follows:
+
+```plaintext
+Server=local.hasura.dev,1433;Database=master;Uid=sa;Password=Password123;TrustServerCertificate=true
+```
+
+:::warning Certificate security
+
+`TrustServerCertificate=true` should only be added for local databases.
+
+:::
+
+### Step 5. Introspect your SQL Server database
+
+```sh title="Next, use the CLI to introspect your SQL Server database:"
+ddn connector introspect my_sqlserver
+```
+
+After running this, you should see a representation of your database's schema in the
+`app/connector/my_sqlserver/configuration.json` file; you can view this using `cat` or open the file in your editor.
+
+```sh title="Additionally, you can check which resources are available, and their status, at any point using the CLI:"
+ddn connector show-resources my_sqlserver
+```
+
+### Step 6. Add your model
+
+```sh title="Now, track the table from your SQL Server database as a model in your DDN metadata:"
+ddn model add my_sqlserver users
+```
+
+Open the `app/metadata` directory and you'll find a newly-generated file: `Users.hml`. The DDN CLI will use this Hasura
+Metadata Language file to represent the `users` table from SQL Server in your API as a
+[model](/reference/metadata-reference/models.mdx).
+
+:::tip
+
+To track all available models in the dababase, run:
+
+```sh
+ddn model add my_sqlserver "*"
+```
+
+:::
+
+### Step 7. Create a new supergraph build
+
+```sh title="To create a local build, run:"
+ddn supergraph build local
+```
+
+The build is stored as a set of JSON files in `engine/build`.
+
+### Step 8. Start your local services
+
+Use the `docker-start` script in the Hasura context to start your local services.
+
+```sh title="Start your local Hasura DDN Engine and SQL Server connector:"
+ddn run docker-start
+```
+
+Your terminal will be taken over by logs for the different services.
+
+### Step 9. Run your first query
+
+```sh title="In a new terminal tab, open your local console:"
+ddn console --local
+```
+
+```graphql title="In the GraphiQL explorer of the console, write this query:"
+query {
+  users {
+    userId
+    name
+    age
+  }
+}
+```
+
+```json title="You'll get the following response:"
+{
+  "data": {
+    "users": [
+      {
+        "userId": 1,
+        "name": "Alice",
+        "age": 25
+      },
+      {
+        "userId": 2,
+        "name": "Bob",
+        "age": 30
+      },
+      {
+        "userId": 3,
+        "name": "Charlie",
+        "age": 35
+      }
+    ]
+  }
+}
+```
+
+### Step 10. Iterate on your SQL Server schema
+
+```sh title="Let's add a new table for posts:"
+CREATE TABLE posts (
+  user_id int,
+  post_id int primary key,
+  title varchar(255),
+  content varchar(255)
+  FOREIGN KEY (user_id) REFERENCES users(user_id)
+);
+```
```

Comment:
For this and the subsequent SQL statements, multi-line input fails on Docker exec. Let's modify this to a one-liner like the initial statements.